### DISCUSSION TITLE
RDS RT/PS history page

### DISCUSSION THREAD

**Discussion by AaronJowitt** [PE5PVB/TEF6686_ESP32#360] [Ideas]
Add a 3rd advanced RDS page, for a RT (vertically scrollable?) history list. Whenever station changes RT, add a new history line at the top containing;

hh:mm:ss RT 

with RT auto horizontal scroll? At least one full page of 'last events' history.  Tuning to a different station clears the list. Optionally, pressing the tune mode button toggles to have each line contain;

hh:mm:ss PS

Would be useful generally & handy for DXing.

**Comment by PE5PVB** [comment]
Why is this usefull for DX'ing?

**Reply by AaronJowitt** [reply]
True, more interesting than useful for DXing. Though for signals that fade up down, RT/PS history may help identify the station if you missed seeing it.  

For general use, particularly when you're not constantly looking/listening, it can be useful to scroll through and identify all songs/events played on a station in the last hour.  Much like the reason some stations publish recently played song info on their websites.

**Comment by andimik** [comment]
Connect to a PC and log with Rdsspy or redsea. I don't like the idea misusing TEF for logging.

**Comment by PE5PVB** [comment]
I see no possible way to intergrate this correctly to the radio. If anyone has a good idea, please explain how it should work in an user-friendly way.
